tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -1101,7 +1101,7 @@ def generate_files(line_list, args):
     files_list.append("<file src=" + '"' + os.path.join(args.sources_path, "README.md") + '" target="README.md" />')
 
     # Process License, ThirdPartyNotices, Privacy
-    files_list.append("<file src=" + '"' + os.path.join(args.sources_path, "LICENSE") + '" target="LICENSE.txt" />')
+    files_list.append("<file src=" + '"' + os.path.join(args.sources_path, "LICENSE.txt") + '" target="LICENSE.txt" />')
```

Comment:
There's no LICENSE.txt file in the source tree. So, not sure what is copied.

Reply:
This [commit](https://github.com/microsoft/onnxruntime/commit/46bd0e63502273b7f5ec9a814d6741eca80784da) should work as seen in this [pipeline](https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=342538&view=results).